k8s/upgrade/src/user_prompt.rs

```diff
@@ -2,10 +2,10 @@
 pub const UPGRADE_WARNING: &str =  "\nVolumes which make use of a single volume replica instance will be unavailable for some time during upgrade.\nIt is recommended that you do not create new volumes which make use of only one volume replica.";
 
 /// Warning to users before doing an upgrade.
-pub const REBUILD_WARNING: &str =  "\nThe cluster is rebuilding replica of some volumes.\nPlease try after some time or skip this validation by retrying with '--ignore-rebuild` flag.";
+pub const REBUILD_WARNING: &str =  "\nThe cluster is rebuilding replica of some volumes.\nPlease try after some time or skip this validation by retrying with '--ignore-replica-rebuild` flag.";
 
 /// Warning to users before doing an upgrade
-pub const SINGLE_REPLICA_VOLUME_WARNING: &str =  "\nThe list below shows the single replica volumes in cluster.\nThese single replica volumes may not be accessible during upgrade.\nTo skip this validation, please retry  with '--skip-single-replica-volume-validation` flag.";
+pub const SINGLE_REPLICA_VOLUME_WARNING: &str =  "\nThe list below shows the single replica volumes in cluster.\nThese single replica volumes may not be accessible during upgrade.\nTo skip this validation, please retry  with '--bypass-single-replica-volume-validation` flag.";
```

Comment:
suggest keeping skip since kubectl also uses skip.

Also seems `--ignore-replica-rebuild` should also be `--skip-replica-rebuild-validation` ?
A question for the future, should this be a rebuild validation or a volume validation, eg: by default only proceed if all volumes are online which effectively means there's no rebuilds?

Reply:
IMHO its a rebuild validation, which actually means that some volume are not healthy and thus rebuild is in progress.
Since we have a single replica volume validation , going with rebuild validations here. Can modify as suggested in other PR.